lib/src/main.dart

```diff
@@ -218,3 +219,26 @@ class MainPage extends StatelessWidget {
     );
   }
 }
+
+/// When thumbnail is available draw image using full width available.
+/// 
+/// TODO: replace thumbnail with hi-res image.
+Widget addImage(Submission submission) {

```

Comment:
We shouldn't need a comment like `/// When thumbnail is available draw image using full width available.`, this should be explained in the code if possible.

something like `Widget addThumbnail` would be enough.

Reply:
Comments help find things faster and understand something that today is obvious after we forget the obvious parts. I'd rather add more comments to the existing code, and break existing functions so it's easier for me to find what I'm looking for.